src/core_plugins/kibana/public/dashboard/listing/dashboard_landing_page_table.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { LandingPageTable } from 'ui_framework/components';
+import { getDashboardColumns } from './get_dashboard_columns';
+import { ItemTableActions } from 'ui/saved_object_table/item_table_actions';
+import { DashboardConstants } from '../dashboard_constants';
+
+export function DashboardLandingPageTable({ itemTableState, onDeleteItems, onFilterItems, isFetchingItems }) {
```

Comment:
(Yellow flag) This view seems light on logic.  What is its responsibility?

Can we give it more responsibilities by filling it with more logic, or making it more responsible for connecting logic to components?